src/current/cockroachcloud/folders.md

```diff
@@ -75,19 +73,9 @@ The following roles allow creation of clusters at the level of the hierarchy whe
 
 The following additional roles explicitly allow management of folders and their contents:
 
-- **Folder Admins** can create, rename, and move, or delete folders where they are granted the role, and they can also manage access to these folders. This role can be granted at the level of the organization or on a specific folder. If granted on a specific folder, the role is inherited by descendant folders.
-
-    A user with the [Org Administrator]({% link cockroachcloud/authorization.md %}#org-administrator) role can grant themselves, another user, or a service account the Folder Admin role.
+- {% include cockroachcloud/org-roles/folder-admin.md %}
 
-    To create or manage clusters in a folder, a Folder Admin also needs the Cluster Administrator or [Cluster Creator]({% link cockroachcloud/authorization.md %}#cluster-creator) role on that folder directly or by inheritance. To delete a cluster, the Cluster Administrator role is required on the cluster directly or by inheritance.
-
-- **Folder Movers** can rename or move descendant folders, and can move clusters within the folder hierarchy where they have the role. However, Folder Movers cannot create or delete folders or clusters, and cannot assign roles. Folder Movers can move clusters within the folder hierarchy even if they do not have a role that allows them to connect to the cluster, such as Cluster Creator or Cluster Operator.
-
-    {{site.data.alerts.callout_info}}
-    A cluster cannot be renamed.
-    {{site.data.alerts.end}}
-
-    A user with the Org Administrator or Folder Admin role can grant another user or a service account the Folder Mover role. Because the Folder Admin role is a superset of Folder Mover, there is no need for a Folder Admin to grant themselves the Folder Mover role.
+- {% include cockroachcloud/org-roles/folder-mover.md %}
 
 The remainder of this page shows how to create folders, manage access to them, and use them to organize your clusters.
 
```

Comment:
Shortly below here there is some recommended security advice that sounds surprising to me.   I agree with limiting the number of users with FOLDER_ADMIN in general.  But the suggestion is to enable it at a sub  folder level which is a bit surprising to me.  It means subfolder management would be a free for all for those users.  I guess that's fine if that's how they want to run their team.  For example "manage it how  you want under this folder".   My recommendation would rather be to have teams communicate their preferences with a single folder manager and just have that person keep their role.  This may be more important   if the org is using folders as a method of dividing up costs.

Reply:
Discussed in Slack the reasons for that advice (mini Org Admin limited to a folder). We should revisit it before GA and see if it is still what we want to recommend.

Reply:
After further discussion, we decided to remove the note.